src/reducers/AddSiteReducer.js

```diff
@@ -75,6 +76,13 @@ export default (state = INITIAL_STATE, action) => {
         case ADD_SITE_FIELDS_RESET:
             return INITIAL_STATE;
 
+        case CURRENT_LOCATION_UPDATED:
+            const {currentLocation: {longitude, latitude}} = payload;
+            const stringLong =  _.toString(longitude);
+            const stringLat = _.toString(latitude);
+
+            return {...state, longitudeText: stringLong, latitudeText: stringLat};
```

Comment:
When we get a successful location, pass it to state so it displays in the `AddSiteScreen` form